content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md

```diff
@@ -45,19 +45,21 @@ It's good practice to structure your files well, so we'll expect you to have a s
 
 The [data set](data.csv) consists of health and demographic data for the period 2014-2015, obtained from [Global Health Observatory Data Repository](http://apps.who.int/gho/data/node.main). Here is some [metadata](data-info.txt) that may be useful.
 
-1. Are there any missing values in the data set? You can deal with missing values by either dropping them or filling them with some value (in a variety of ways) or ignoring them. What do you think is best in this case? And why? Here is a resource for dealing with missing values : https://www.analyticsvidhya.com/blog/2021/10/a-complete-guide-to-dealing-with-missing-values-in-python/
+1. Load the dataset into a pandas DataFrame. Are there any missing values in the dataset? You can deal with missing values by either dropping them or filling them with some value (in a variety of ways depending on the kind of data). Later in this program you'll be using learning how to train machine learning models using various kinds of data. When doing so, missing values simply cannot be ignored. Further, when it comes to datasets with many columns (or features), it will also not do to drop an entire row for a few missing values. 
```

Comment:
"you'll be using learning how to train...". Some of these sentences need to be cleaned up.

Reply:
hmm... 

You ask: "what is the best approach in this case for dealing with missing values?". Is this rhetorical? Or are the learners meant to answer the question in their notebook? 

1. load the dataset
2. deal with the missing values. Maybe store it in a dataframe with a name at this point. We can then check the row count and the values used and test people on understanding the missing values thing. I also think it is worth trying to make the text about missing data a bit more cohesive, it's strange that there is one unordered list element with some into in, and the main point has the rest of the info
3. deal with any other problems. Store it in a dataframe with a name. Run more tests 
...

Reply:
Yes, def some grammar clean up, working too fast, etc

So I think that we do want them to have to reason out why they chose the method they chose and answer this in their notebook  and encourage them to go look at the resource provided. I suppose learners don't do this sometimes and maybe it is to do with the structure seeming a bit rhetorical, lol, so ja, perhaps more structure will help.

My main concern is that I don't want to tell them fully what to do upfront, but they need to replace missing values with a regionally determined metric for that feature, either mean or median